src/components/Element/ArticleElement/Calendar.jsx

```diff
@@ -0,0 +1,3 @@
+export function Calendar() {
```

Comment:
you should stash files that aren't used yet and commit/push them later. Production code should be as clean as possible.